src/components/socialMedia/socialMedia.css

```diff
@@ -5,9 +5,12 @@
     justify-content: center;
     align-items: center;
     flex-direction: row;
-    border: 2rem black solid;
+    border: 2rem rgb(36, 36, 36) solid;
     border-bottom-width: 4rem;
     border-radius: 2rem;
+    background-color: rgb(255, 247, 223);
```

Comment:
For me personally, the background color looks fine. However, if you want to check for accessibility later down the line, I'd recommend building a sample deployable that opens up all sections and checking if our current color scheme meets accessibility requirements through this website!
 > https://ace.accessibe.com/?utm_source=o&utm_medium=e&utm_campaign=u
 
I tried running it through the checker rn, but it doesn't do a complete check since not the entire case study is open at first.